lib/cadet_web/controllers/assessments_controller.ex

```diff
@@ -0,0 +1,163 @@
+defmodule CadetWeb.AssessmentsController do
+  use CadetWeb, :controller
+
+  use PhoenixSwagger
+
+  swagger_path :index do
+    get("/assessments")
+
+    summary("Get a list of all assessments")
+
+    security([%{JWT: []}])
+
+    produces("application/json")
+
+    response(200, "OK", Schema.ref(:AssessmentsList))
+    response(401, "Unauthorised")
+  end
+
+  swagger_path :show do
+    get("/assessments/{assessmentId}")
+
+    summary("Get information about one particular assessment.")
+
+    security([%{JWT: []}])
+
+    produces("application/json")
+
+    parameters do
+      assessmentId(:path, :integer, "assessment id", required: true)
+    end
+
+    response(200, "OK", Schema.ref(:Assessment))
+    response(400, "Missing parameter(s) or invalid assessmentId")
+    response(401, "Unauthorised")
+  end
+
+  def swagger_definitions do
+    %{
+      AssessmentsList:
+        swagger_schema do
+          description("A list of all assessments")
+          type(:array)
+          items(Schema.ref(:AssessmentOverview))
+        end,
+      AssessmentOverview:
+        swagger_schema do
+          properties do
+            id(:integer, "The assessment id", required: true)
+            title(:string, "The title of the assessment", required: true)
+            type(:string, "Either mission/sidequest/path/contest", required: true)
+            summary_short(:string, "Short summary", required: true)
+            open_at(:string, "The opening date", format: "date-time", required: true)
+            close_at(:string, "The closing date", format: "date-time", required: true)
+
+            max_xp(
+              :integer,
+              "The maximum amount of XP to be earned from this assessment",
+              required: true
+            )
+
+            cover_picture(:string, "The URL to the cover picture", required: true)
+          end
+        end,
+      Assessment:
+        swagger_schema do
+          properties do
```

Comment:
I think we should add `category` here just to standardise with the frontend data structure

Reply:
I didn't add this cos `AssessmentCategory` was in a different colour in the .ts schema. Need to get my eyes checked

Reply:
Lmao at least the commit message isn't `I'm blind`